sklearn/ensemble/forest.py

```diff
@@ -913,6 +913,12 @@ class RandomForestClassifier(ForestClassifier):
         Note that these weights will be multiplied with sample_weight (passed
         through the fit method) if sample_weight is specified.
 
+    increasing : list of ints, optional (default=None)
+        Indices of features to have a monotonically increasing effect.
```

Comment:
I think the relationship between the how this constraint is specified and what its effect is is unclear from this description.

Reply:
Indices of features to have a monotonically increasing effect.

=>

Indices of features to constrain to have a monotonically increasing effect on the predicted variable.